hooks/post/10-post_install.rb

```diff
@@ -37,7 +37,7 @@ def new_install?
       say <<MSG
   * To install additional capsule on separate machine continue by running:
 
-      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "<%= color('~/$CAPSULE-certs.tar', :info) %>"
+      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "<%= File.join('/root', File.basename('certs.tar.gz')) %>"
```

Comment:
Actually, this is just the help, just leave the original suggestion, or make it a hard coded path, although the above comment applies to the other reference